app/assets/javascripts/availabilities.js.coffee

```diff
@@ -0,0 +1,6 @@
+jQuery ->
+  $('#availabilities').dataTable
+    order: [[0, 'desc'], [1, 'asc']],
+    columnDefs: [
```

Comment:
Is this enough for `columnDefs` ? While we have a hidden column now, I believe clicking on the column marked `Title` will still sort by that column. I think you need something like => `{ orderData: 2, targets: 3 }`
You can see that here ->
https://github.com/ReadyResponder/ReadyResponder/blob/development/app/assets/javascripts/events.js.coffee#L5

Reply:
I think you are right. orderData , targets will allow sorting by rank when user clicks title column (instead of the title word(s) itself), worthwhile if there is no use case for sorting by title names